service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisGeneratedOrder.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType;
+
+@Getter
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+public class DocmosisGeneratedOrder extends DocmosisOrder {
```

Comment:
I think this can be covered by this ticket. https://tools.hmcts.net/jira/browse/FPLA-1793 The Docmosis class hierarchy needs to addressed as a whole. I think there should be DocmosisBlankOrder, DocmosisEPO etc which extend DocmosisOrder.

Reply:
Sounds good to me, maybe make a note of that in the ticket

Reply:
Will do